streams/ru.m3u

```diff
@@ -29,7 +29,7 @@ http://dancehits80.denvs.ru:8080/index.m3u8
 http://sirius.greenhosting.ru/Kino24Ru/video.m3u8
 #EXTINF:-1 tvg-id="",Kulinar TV Show
 https://api.alpaca.t62a.com/hls/9110/index.m3u8
-#EXTINF:-1 tvg-id="",MCM Top
+#EXTINF:-1 tvg-id="MCMTop.ru",MCM Top
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="MCMTop.ru",MCM Top (Russia)
```

Since MCM is French and they still have distribution rights in Russia (if French people knew...) better make the precision this is the Russian version.